pkgs/development/tools/doctl/default.nix

```diff
@@ -2,15 +2,24 @@
 
 buildGoPackage rec {
   name = "doctl-${version}";
-  version = "1.3.1";
-  rev = "a57555c195d06bc7aa5037af77fde0665ad1231f";
+  version = "1.5.0";
+  rev = "66f10023539b5e0e27485a34048366ca1133872f";
   goPackagePath = "github.com/digitalocean/doctl";
 
+  excludedPackages = "cmd/doctl-gen-doc cmd/install-doctl cmd/release-doctl";
+  buildFlagsArray = let t = "${goPackagePath}"; in ''
+     -ldflags=
+        -X ${t}.Major=1
```

Comment:
you could make variables `major`, `minor` and `patch` and use those to build `version` and `ldflags`